test/test_db_connection.py

```
@@ -1,13 +1,13 @@
-
-test_connection()
+from db import SessionLocal
+from sqlalchemy import text
+
+def test_connection():
+    try:
+        db = SessionLocal()
+        db.execute(text("SELECT 1")) 
+        db.close()
+        print("✅ Database connection successful!")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)